02week/blog/index.html

```diff
@@ -0,0 +1,111 @@
+        </p>
+          <ol>
+            <li>
+              Developers seem to have endless possibilities of learning and opportunities for creative problem-solving.
+            </li>
+            <li>
+               In the web developing community, there is an openness of sharing knowledge.
+            </li>
+            <li>
+              Working on projects and collaborating with others is rewarding.
+            </li>
+            <li>
+              Developers are paid well.
+            </li>
+          </ol>
+      </div>
+      <div>
+        <h2>How Being a Teacher May Help Me in the Web Development World</h2>
+        <p>
+          I can't help from thinking about all the incredible experiences that I have had as a music teacher, and how much of these skills could translate into being a developer:
+        </p>
+          <ol>
+            <li>
+              The ability to communicate effectively and percisely when describing my own thought process, and willingness to learn other people's thought processes.
+            </li>
+            <li>
+              Being able to organize and create systems for projects on time frames.
+            </li>
+            <li>
+              "Keep it Simple, Silly" is a common phrase I use in my classroom when teaching students about creating compositions. While complex music is really neat, if no one can read it; there's no point.
+            </li>
+            <li>
+              Believing in the big picture and commiting to the process, even if that means having to work a little bit more than planned.
+            </li>
+            <li>
+              Staying positive by looking for ways to grow and learn.
+            </li>
+            <li>
+              The ability to improvise solutions for problems on the spot.
+            </li>
+          </ol>
+        <p>
+          I have taught English, Math, and Music in many different school settings. Currently I teach Prek-8th Grade students music at a private school in Austin.
+        </p>
+      </div>
+      <div>
+        <h2>I Love CSS!!!</h2>
+        <p>
+          We are in our second week of learning how to use HTML and CSS. It has been so much fun testing out different styles and watching as the page changes like magic before your eyes! It's even fun when something doesn't work because it becomes this fun puzzle that you have to solve. I can't believe this is part of my homework, check out what I can do already:
+        </p>
+          <ol>
+            <li id="green">
+              I can make this line green by using an id
+            </li>
+            <li id="border">
+              And this line with a pretty border
+            </li>
+            <li class="container">
+              And put your curser over this image
```

Comment:
Try to use semantic tags when you can, even for simple text. I.e. wrap this in a `p` tag